### ISSUE TITLE
Add option to use Thaliak's Favor as necessary to be able to use Patience I/II

### ISSUE THREAD

**Issue by retep998** [InitialDet/AutoHook#139]
When trying to catch fish using Patience it's possible to get into weird situations my GP is just not high enough to use Patience II while still being over the threshold for cordial (because that would overcap me) while also having no Angler's Art stacks available to squeeze out that extra bit of GP (because they're constantly being used up). It would be nice if I could have a separate threshold for Thaliak's Favor where it holds on to *some* Angler's Art stacks to save for those situations where I want to use Patience II but don't have enough GP. It should still of course spend Angler's Art on Thaliak's Favor to prevent overcapping Angler's Art (and reduce cordial usage).

So basically:
* Lower Angler's Art threshold: use Thaliak's Favor above this amount *only* when needed for Patience.
* Higher Angler's Art threshold: use Thaliak's Favor above this amount to prevent overcapping.
